selectrum.el

```diff
@@ -1129,6 +1151,23 @@ defaults to the current one and MAX which defaults to
         (max (window-body-height window) n)
       n)))
 
+(defun selectrum--preprocess (candidates)
+  "Preprocess CANDIDATES list.
+The preprocessing applies the `selectrum-preprocess-candidates-function'
+and the `x-group-function'."
+  (setq selectrum--preprocessed-candidates
```

Comment:
Can be changed to `setq-local`

Reply:
Fixed now. I missed this comment before.